src/node_http_common.h

```diff
@@ -261,7 +261,7 @@ class NgRcBufPointer : public MemoryRetainer {
  public:
   typedef typename T::rcbuf_t rcbuf_t;
   typedef typename T::vector_t vector_t;
-  NgRcBufPointer() {}
+  NgRcBufPointer() : buf_(nullptr) {}
```

Comment:
Can this be `buf_ = nullptr;` in the field definition instead? It’s cleaner and avoid theses changes here